pkgs/by-name/ta/tana/package.nix

```diff
@@ -0,0 +1,114 @@
+{ xorg
+, stdenv
+, lib
+, alsa-lib
+, at-spi2-atk
+, atkmm
+, cairo
+, cups
+, dbus
+, expat
+, glib
+, gtk3
+, libdrm
+, libglvnd
+, libxkbcommon
+, mesa
+, nspr
+, nss
+, pango
+, systemd
+, fetchurl
+, autoPatchelfHook
+, dpkg
+, makeWrapper
+, wrapGAppsHook
+}:
+let
+  xorgLibs = with xorg; [
+    libX11
+    libxcb
+    libXcomposite
+    libXdamage
+    libXext
+    libXfixes
+    libXrandr
+  ];
+  libs = [
+    alsa-lib
+    at-spi2-atk
+    atkmm
+    cairo
+    cups
+    dbus
+    expat
+    glib
+    gtk3
+    libdrm
+    libglvnd
+    libxkbcommon
+    mesa
+    nspr
+    nss
+    pango
+  ];
+  buildInputs = xorgLibs ++ libs;
```

Comment:
Why are those split?

Reply:
To use the `with` clause (I was importing the whole `xorg` in my derivation), but in the new version I am pulling only the needed libraries from directly